src/routes/index.js

```diff
@@ -1,4 +1,8 @@
 const express = require("express");
 const router = express.Router();
+const sign = require("../middlewares/sign");
+
+const cartRouter = require("./cartRouter");
+router.use("/cart", sign.myCustomMiddleware, cartRouter.router);
```

Comment:
root target은 복수로 만들어주는 것이 일반적입니다.
```suggestion
router.use("/carts", sign.loginRquired, cartRouter.router);
```